web/src/components/ExpandedDetail.js

```diff
@@ -65,15 +65,8 @@ class ExpandedDetail extends React.Component {
     //let api = urljoin('https://' + window.location.host, '/releases/', this.props.data.name, 'resources') // point to local IP for testing
     let api = 'http://localhost:8080/resources'
     axios.get(api).then(response => {
-      var pods = []
-      var resources = []
-      for (var i = 0; i < response.data.length; i++) {
-        if (response.data[i].kind === "Pod") {
-          pods.push(response.data[i])
-        } else {
-          resources.push(response.data[i])
-        }
-      }
+      let pods = response.data.filter((d) => d.kind === 'Pod')
```

Comment:
We could actually just put these lines inside the setState call below.

Reply:
Fixed.